Disable TestAttach

### Description

Despite many attempts at tweaking timeout, TestAttach keeps on failing.

I do have a rewritten version that seems to flake less, but there is still a lot of work on containers tests before I can send it.

Suggesting we skip it for the time being for quality of life.

### Steps to reproduce the issue

Run tests.

### Describe the results you received and expected

```
=== RUN   TestAttach
=== PAUSE TestAttach
=== CONT  TestAttach
    container_attach_linux_test.go:58: assertion failed: 
        Command:  /usr/bin/unbuffer -p /usr/local/bin/nerdctl --namespace=nerdctl-test run -it --name nerdctl-testattach ghcr.io/stargz-containers/alpine:3.13-org
        ExitCode: 0
        Stdout:   
        Stderr:   
        
        Failures:
        Expected stdout to contain "read detach keys"
--- FAIL: TestAttach (7.03s)
```

### What version of nerdctl are you using?

main

### Are you using a variant of nerdctl? (e.g., Rancher Desktop)

None

### Host information

_No response_

--- a/cmd/nerdctl/container/container_attach_linux_test.go
+++ b/cmd/nerdctl/container/container_attach_linux_test.go
@@ -63,6 +63,8 @@ func prepareContainerToAttach(base *testutil.Base, containerName string) {
 func TestAttach(t *testing.T) {
 	t.Parallel()
 
+	t.Skip("This test is very unstable and currently skipped. See https://github.com/containerd/nerdctl/issues/3558")
+
 	skipAttachForDocker(t)
 
 	base := testutil.NewBase(t)

